### ISSUE TITLE
function get_dimensions is only available in the nightly version

### ISSUE THREAD

**Issue by hukkai** [pytorch/vision#5944]
### 🐛 Describe the bug

**Describe the bug**
In the reference codes for training ImageNet classification, function `torchvision.transforms.functional.get_dimensions` is used here:
https://github.com/pytorch/vision/blob/62740807c18e68bb0acd85895dca527f9a655bd5/references/classification/transforms.py#L152

However, as I update my torch vision version to the latest one `0.12.0`, I cannot find this function. This function is available at version `0.13.0.dev20220502`.

Besides, is it possible to provide training logs for the method (ResNet50 80.858 top1 accuracy on ImageNet) mentioned in this blog (https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/)? Thanks!

### Versions

**Versions**

Collecting environment information...
PyTorch version: 1.11.0
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: macOS 11.6.4 (x86_64)
GCC version: Could not collect
Clang version: 13.0.0 (clang-1300.0.29.30)
CMake version: Could not collect
Libc version: N/A

Python version: 3.8.1 (default, Jan  8 2020, 16:15:59)  [Clang 4.0.1 (tags/RELEASE_401/final)] (64-bit runtime)
Python platform: macOS-10.16-x86_64-i386-64bit
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.22.3
[pip3] pytorchvideo==0.1.5
[pip3] torch==1.11.0
[pip3] torchvision==0.12.0
[conda] blas                      1.0                         mkl
[conda] mkl                       2021.4.0           hecd8cb5_637
[conda] mkl-service               2.4.0            py38h9ed2024_0
[conda] mkl_fft                   1.3.1            py38h4ab4a9b_0
[conda] mkl_random                1.2.2            py38hb2f4e1b_0
[conda] numpy                     1.19.5                   pypi_0    pypi
[conda] pytorchvideo              0.1.5                    pypi_0    pypi
[conda] torch                     1.11.0                   pypi_0    pypi
[conda] torchvision               0.12.0                   pypi_0    pypi

**Comment by pmeier** [comment]
> Besides, is it possible to provide training logs for the method (ResNet50 80.858 top1 accuracy on ImageNet) mentioned in this blog (https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/)

cc @datumbox
